anima-jws-voucher.mkd

```diff
@@ -1,7 +1,7 @@
 ---
 title: JWS signed Voucher Artifacts for Bootstrapping Protocols
 abbrev: JWS-voucher
-docname: draft-ietf-anima-jws-voucher-12
+docname: draft-ietf-anima-jws-voucher-13
```

Comment:
Just to comment: with my simple makefiles, best to always bump the version number in a separate PR/commit, otherwise, it becomes a useless conflict.